lib/skate/detours/detours.ex

```diff
@@ -95,19 +95,25 @@ defmodule Skate.Detours.Detours do
 
   @type detour_type :: :active | :draft | :past
 
+  @doc """
```

Comment:
Non-blocking, but wouldn't hate this as a fast-follow: This is a good candidate for doctests.

Reply:
Let's not block this PR on that - I'll queue it up as a fast-follow and see if I can get it done in the 16 hours remaining to me.